### ISSUE TITLE
Stability Boots Proc + Mob autobash in Ashstone

### ISSUE THREAD

**Issue by sekfarok** [torilmud/issues#192]
Ok here it is, we've tried to kill Lady Christine in Ashstone. What happen was, my boots procs autostand everytime i got bashed. But the mob (timid deer + tuskmite) autobash  with !fail every time i stand. It causes me stuck and unable to send any command to the MUD.

Please can someone look into the issue.

- Teguh -

**Comment by shevarash** [comment]
That's not good. We'll have to update the stability proc.

**Comment by cmoncrief** [comment]
What mob exactly is bashing you? The Timid Deer and Tuskmite mobs don't appear to have procs.

**Comment by shevarash** [comment]
What mob exactly is bashing you? The Timid Deer and Tuskmite mobs don't appear to have procs.

**Comment by marthammor** [comment]
They both have powers bash assigned to them.

**Comment by shevarash** [comment]
Is that intentional?

**Comment by marthammor** [comment]
I believe so. They are assigned in the mob file along with some templates on some mobs.

**Comment by ghost** [comment]
there's a couple places in ashstone, where you will actually lose link for lack of commands, because of stability boots and bash, either the stability stand needs to have 0 lag, i'm sure that stability boots making you stand up is good, but when you still have the same lag as if you didn't have the boots on, it's kind of pointless, because in multiple mob fights, it just opens you up to more bashing/tripping, rather than mobs trying to kick you, which can actually miss, rather than bash/trip which rarely misses.  ashstone isn't the only place this loop exists, but it is the worst, with powers bash, which seems to have 0 save for players.  another place where you can really see just how terrible the boots lag is, is the undead section of um2 where you get hammer head for torin, 8 mobs in room, you're lucky to get a command to go thru in 5 minutes because of bashing putting you in stability boots lag.

**Comment by ghost** [comment]
also, the avoid bash/riposte attack on stability boots lagging you is another issue, unlike just being lucky and the bash missing, or a standard riposte in combat, the proc gives you lag, which adds to the infinite lag you obtain from large group fights, i love the procs on boots, but the lag from them is the issue, and if removed, would solve so many of these loops, but at the same time, wouldn't totally change fight dynamics a lot, because a group of 10 mobs will still kick lag you into oblivion

**Comment by GeneralGormal** [comment]
I'm late to the stability-hate party, but I'm here now.  I quit using these boots entirely on my tanks because the auto-stand makes you a valid bash/kick target again and you get lagged to infinity any time you're fighting large groups.  The extra attack you get from avoiding/standing is nice, but as long as it doesn't prevent the delay portion of being bashed, you might as well just be bashed.

TLDR: Stability boots are flat out bad for tanking because of the delay stacking.

**Comment by beshaba-toril** [comment]
IIRC, that used to (well before powers) happen all the time in Izans on
entry fight and SF with kicking skellies. The bash/kick lag stacks.

Always prayed that you've got enough mobs alive to keep Valhalla blur up.

-- Besh

**Comment by torilmud-silvanus** [comment]
@shevarash did you have an areas solution you wanted applied to this issue?
